src/module.ts

```diff
@@ -634,8 +634,8 @@ export default defineNuxtModule<ModuleOptions>({
     // @nuxtjs/tailwindcss support
     // @ts-ignore - Module might not exist
     nuxt.hook('tailwindcss:config', (tailwindConfig) => {
-      tailwindConfig.content = tailwindConfig.content ?? []
-      tailwindConfig.content.push(resolve(nuxt.options.buildDir, 'content-cache', 'parsed/**/*.md'))
+      tailwindConfig.content = tailwindConfig.content ?? [];
+      (Array.isArray(tailwindConfig.content) ? tailwindConfig.content : tailwindConfig.content.files).push(resolve(nuxt.options.buildDir, 'content-cache', 'parsed/**/*.md'))
```

Comment:
Are we sure that `tailwindConfig.content.file` always exists if `tailwindConfig.content` is an object? If not, we should handle `undefined` state of `tailwindConfig.content.file`

Reply:
I'm confident that it will exist (based on type definition and custom config merging strategy in the module), but better to be safe than sorry - made the change!